src/utils.js

```diff
@@ -397,6 +397,8 @@ function isNot(type) {
 }
 
 export function generateFriendlyStringFromSubmissionFormats(submissionType) {
+  if (submissionType === null) return null;
```

Comment:
Shouldn't this return `submissionTypeLabels[submissionTypes.NONE]`?

Reply:
That was my initial thought, but using NONE it renders this:

**Submitting**: Nothing

IMHO we don't want to declare that this assignment has nothing to submit, maybe we just don't know that is submitted, and we what to not render the `Submitting` section. This is what we get when passing an unknown type, like 'n/a'.